README.md

```diff
@@ -69,3 +75,78 @@ When the Django server is running, you can start the Django shell with:
 Run the back-end test suite with:
 
     docker-compose exec backend pipenv run python manage.py test --settings=donate.settings_test
+
+
+## Basket
+
+[Basket](https://github.com/mozmeao/basket) is a tool run by MoCo to manage newsletter subscriptions and donations. It's listening for messages (JSON) sent to a SQS queue.
+
+### Basket donations queue:
+
+Basket has 4 [donation event types](https://github.com/mozmeao/basket/blob/master/basket/news/management/commands/process_donations_queue.py#L21-L26): 
+
+- `donation`: process a donation and send its data to SFDC,
+- `crm_petition_data`: add petition signature to SFDC,
+- `newsletter_signup_data`: newsletter signup for the foundation site,
+- `DEFAULT`: process a followup Stripe event on a donation.
+
+For this project, we're only using the `donation` event type.
+
+### Donation event type
+
+Example of a donation message sent to Basket:
+
+```
+{ event_type: 'donation',
+  last_name: 'alex',
+  email: 'alex@alex.org',
+  donation_amount: 50,
+  currency: 'usd',
+  created: 1563801762,
+  recurring: false,
+  service: 'stripe',
+  transaction_id: 'ch_1Ez1TSG8Mmx3htnxyShib70n',
+  project: 'mozillafoundation',
+  last_4: '4242',
+  donation_url: 'http://localhost:3000/en-US/',
+  locale: 'en-US',
+  conversion_amount: 50,
+  net_amount: 48.6,
+  transaction_fee: 1.4 }
+```
+
+`event_type`: Basket event type. Should be donation,
+`first_name`: first name of the donor (optional),
+`last_name`: last name of the donor,
+`email`: email of the donor,
+`donation_amount`: amount of the donation,
+`currency`: letter code for the currency,
+`created`: unix timestamp,
+`recurring`: `false` for a one-time donation or `true` for a recurring one,
+`service`: name of the payment processor (ex: `stripe` or `paypal`)
+`transaction_id`: ID generated by the payment processor,
+`project`: name of the project that will receive the donation (ex: `thunderbird`, `mozillafoundation`)
+`last_4`: last 4 digits of the credit card,
+`donation_url`: url from which the donation was made,
+`locale`: language code for the donor,
+`conversion_amount`: donation amount in USD, before transaction fees,
+`net_amount`: donation amount in USD, after transaction fees,
+`transaction_fee`: payment processor's transaction fees in USD
+
+
+### Newsletter signup
+
+We're using Basket newsletter API to signup people to our newsletter. Specs are available in [Basket's documentation](https://basket.readthedocs.io/newsletter_api.html#news-subscribe)
```

Comment:
```suggestion
We're using Basket newsletter HTTP API to signup people to our newsletter. Specs are available in [Basket's documentation](https://basket.readthedocs.io/newsletter_api.html#news-subscribe). While we could use the SQS, the Basket team has expressed interest in moving towards standard HTTP requests for all Basket interaction. (A donation event endpoint will be added in the future.)
```

Reply:
Same as the previous comment: let's not take about possible implementation in the future.

Reply:
```suggestion
We're using Basket newsletter HTTP API to signup people to our newsletter. Specs are available in [Basket's documentation](https://basket.readthedocs.io/newsletter_api.html#news-subscribe). (Note that this is different than the SQS approach used for donation events) 
```